testing.go

```diff
@@ -140,28 +149,72 @@ func stdIOHandler (test *Test, stdio *StdIO, isError bool) {
 	}
 }
 
+// Handles the StdIO for a logger
+func loggerStdIOHandler(test *Test, stdio *StdIO) {
+	stdio.sync.Add(1)
+	defer stdio.sync.Done()
+	reader := bufio.NewReader(stdio.readPipe)
+	for {
+		line, err := reader.ReadString('\n')
+		if err != nil {
+			break
+		}
+		nLine := line
+		flags := log2.Flags()
+		if flags&(log2.Ldate|log2.Ltime|log2.Lmicroseconds) != 0 {
```

Comment:
Is this the best way to parse the timestamp in go?

Reply:
I'm not parsing the time right now, I'm just slicing to remove that part, I base the code in the way the logger format that date: 

https://github.com/golang/go/blob/869620bbeaef7698ef6210c896f913257fe2e43b/src/log/log.go#L98-L124 

I have to check the flags because the logger could be configured differently.

What I'm gonna do is to reduce the slicing with a counter in the if's.